openvino_wrapper_lib/include/openvino_wrapper_lib/inferences/human_pose_estimation.hpp

```diff
@@ -0,0 +1,172 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @brief A header file with declaration for HumanPoseEstimation Class
+ * @file vehicle_attribs_detection.hpp
+ */
+#ifndef OPENVINO_WRAPPER_LIB_INFERENCES_VEHICLE_ATTRIBS_DETECTION_HPP_
+#define OPENVINO_WRAPPER_LIB_INFERENCES_VEHICLE_ATTRIBS_DETECTION_HPP_
+#include <rclcpp/rclcpp.hpp>
+#include <memory>
+#include <vector>
+#include <string>
+#include "openvino_wrapper_lib/models/human_pose_estimation_model.hpp"
+#include "openvino_wrapper_lib/engines/engine.hpp"
+#include "openvino_wrapper_lib/inferences/base_inference.hpp"
+#include "openvino_wrapper_lib/inferences/openpose_decoder.hpp"
+#include "opencv2/opencv.hpp"
+// namespace
+
+
+
+namespace openvino_wrapper_lib
+{
+/**
+ * @class HumanPoseEstimationResult
```

Comment:
make all comments correct and aligh with the code.

Reply:
Fixed.